src/XrdHttp/XrdHttpMon.cc

```diff
@@ -11,7 +11,10 @@ XrdSysError eDest(0, "HttpMon");
 
 std::array<std::array<XrdHttpMon::HttpInfo, XrdHttpMon::StatusCodes::sc_Count>, XrdHttpReq::ReqType::rtCount> XrdHttpMon::statsInfo{};
 
-XrdHttpMon::XrdHttpMon(XrdSysLogger* logP, XrdXrootdGStream* gStream) : gStream(gStream) { eDest.logger(logP); }
+XrdHttpMon::XrdHttpMon(XrdSysLogger* logP, XrdXrootdGStream* gStream) : gStream(gStream) {
+    eDest.logger(logP);
+    flushPeriod = std::chrono::seconds(gStream->GetAutoFlush());
```

Comment:
Hello @bbockelm 
Is this a good way to set the flush period? Since this is not under the `http` directive, and I did not want to parse the mongstream here.

This required adding Getters below.